### PR TITLE
chore: bump otelcol-sumo to the officially supported version v0.47.0-sumo-0

### PR DESCRIPTION
**Pull request by perk-sumo**
Bump otelcol to https://github.com/SumoLogic/sumologic-otel-collector/releases/tag/v0.47.0-sumo-0

### CODE REVIEW

**Review by sumo-drosiek — APPROVED** [review]

**Comment by sumo-drosiek** [comment]
What about tracing images?

**Comment by perk-sumo** [comment]
Tracing AFAIK will move with v3.

File: CHANGELOG.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - chore(deps): bump prometheus node exporter tag to 2.3.1 [#2177][#2177]
 - chore: upgrade Fluentd to 1.14.5-sumo-1 [#2196][#2196]
 - chore: upgrade Falco Helm Chart to 1.17.4 [#2197][#2197]
-- chore: bump sumo ot distro to 0.0.58-beta.0 [#2213][#2213]
+- chore: bump sumo ot distro to 0.0.58-beta.0 [#2220][#2220]
```
Comment (swiatekm-sumo) [inline CHANGELOG.md:25]:
```suggestion
- chore: bump sumo ot distro to 0.47.0-sumo-0 [#2220][#2220]
```

**Review by swiatekm-sumo — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (perk-sumo) [reply CHANGELOG.md:25]: Wait - where does that come from? I have that change in my view 🤔

Comment (perk-sumo) [reply CHANGELOG.md:25]: Ah, I can see it now, thanks :)

**Review by swiatekm-sumo — APPROVED** [review]

**Pull request merged**